src/chain.js

```diff
@@ -3,6 +3,8 @@ var ganacheLib = require("ganache-cli")
 var path = require("path")
 var Web3 = require("web3")
 
+import * as pkg from '../../../package.json'
```

Comment:
When I run `npm start` on this branch, startup hangs. When I check the logs, I get: `Error: Blockchain process exited prematurely with code '1', due to signal 'null'.`

After adding some extra logging to dump errors from the forked subprocess, I get the following:

```
SyntaxError: Unexpected token import
    at Object.exports.runInThisContext (vm.js:78:16)
    at Module._compile (module.js:543:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:424:7)
    at startup (bootstrap_node.js:147:9)
    at bootstrap_node.js:544:3
```